app/helpers/content_helper.rb

```diff
@@ -61,7 +61,7 @@ def progress_indicator(status)
       when :started then 'yellow'
       end
 
-    govuk_tag(text: t(status, scope: 'module_indicator'), colour: colour)
+    govuk_tag(text: t(status, scope: 'module_indicator').capitalize, colour: colour)
```

Comment:
The prototype link you shared @martikat has lowercase, has this changed?
<img width="136" alt="Screenshot 2024-06-25 at 11 21 43" src="https://github.com/DFE-Digital/early-years-foundation-recovery/assets/3261/8266dacb-d647-4b7e-9f46-058a52b75d5a">

Reply:
I have removed this so it matches with the prototype. This can be added in though if we don't want this to be all in lowercase (cc @instantrick )